SwrveDemo/Podfile

```diff
@@ -0,0 +1,5 @@
+
+target 'SwrveDemoFramework' do
+pod 'SwrveSDK', '~> 4.4.0'
```

Comment:
the point is to not target local path so they're using the same tools as we are.